### PR TITLE
Update 22-introductions.md

### PR DESCRIPTION
**Pull request by raynamharris**
A a recent workshop, I had only 30 minutes to deliver the content and give the students an activity to end on a high note (yea for the peak rule). To shorten the existing exercise from 35 minutes to 8 minutes I omitted having the learners give each other feedback. 

Please let me know if you think its good to have both challenges present in the lesson. cc @tracykteal

### CODE REVIEW

**Comment by raynamharris** [comment]
As a side note, this exercise received a positive review during one up one down. I know the main goal is to get people to think about introductions, but I think it also 1) inspires extra confidence in their ability to teach and 2) helps build community by helping the learners get to know each other better.

File: _episodes/22-introductions.md [hunk 1]
```diff
@@ -271,14 +271,23 @@ Let them know about any logistics for the days - lunch times, breaks,
...
 > 
 > This exercise should take 35 minutes.  
 {: .challenge}
 
+> ## Practice Your Introduction without Feedback
+>
+> 1. Imagine you have completed instructor training and you are about to teach a full lesson around the material you have been practicing teaching today. 
+> 2. Rehearsing your introduction in your mind. (3 minutes)
+> 3. Return to your groups of 2 or 3 and give 90 second introductions. (5 min)
+>
```
Comment (ChristinaLK) [inline _episodes/22-introductions.md]: +4. (optional, if there is time) Offer each other feedback.

File: _episodes/22-introductions.md [hunk 2]
```diff
@@ -271,14 +271,23 @@ Let them know about any logistics for the days - lunch times, breaks,
 accessibility, etc.
 
 
-> ## Practice Your Introduction
+> ## Practice Your Introduction with Feedback
 >
```
Comment (ChristinaLK) [inline _episodes/22-introductions.md]: Remove this version of the challenge.

**Comment by tracykteal** [comment]
Seems fine to do it without feedback. I like Christina's suggestion of having the (optional, if there's time) as number 4.  Since it is at the end and we want to end on a high note, maybe line 4 could be, (optional, if there is time) Let the person who presented know what you liked about their introduction.

**Comment by raynamharris** [comment]
Great! Thanks for the suggestions @ChristinaLK @tracykteal. I like having one challenge with an optional "what did you like" discussion

**Comment by ErinBecker** [comment]
Looks great! Thanks @raynamharris!

**Pull request merged**